arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,319 @@
+                "limit": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Gets the maximum size of the database (see 'unit' for the units)."
+                },
+                "unit": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the units that the limit is expressed in.",
+                    "enum": [
+                        "Megabytes",
+                        "Gigabytes",
+                        "Terabytes",
+                        "Petabytes"
+                    ],
+                    "x-ms-enum": {
+                        "name": "MaxSizeUnits",
+                        "modelAsString": true
+                    }
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the status of the maximum size capability.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus",
+                        "modelAsString": true
+                    }
+                }
+            },
+            "description": "Represents the maximum size limits for an Azure SQL Database."
+        },
+        "ServiceObjectiveCapability": {
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the Service Objective name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the status of the Service Objective.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus",
+                        "modelAsString": true
+                    }
+                },
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the unique ID of the Service Objective."
+                },
+                "supportedMaxSizes": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/MaxSizeCapability"
+                    },
+                    "description": "Gets the list of supported maximum Azure SQL Database sizes for this Service Objective."
+                }
+            },
+            "description": "Represents the Service Objectives capabilities."
```

Comment:
As a non-SQL Azure individual, I'd appreciate some additional conceptual information about what Service Objective capabilities are here. In general, it is pretty clear the class FooBar represents a FooBar - ideally we add a bit more value to the user than that in our documentation :)